test/parallel/test-child-process-exec-stdout-stderr-data-string.js

```diff
@@ -4,20 +4,11 @@ const common = require('../common');
 const assert = require('assert');
 const exec = require('child_process').exec;
 
-let stdoutCalls = 0;
-let stderrCalls = 0;
-
 const command = common.isWindows ? 'dir' : 'ls';
-exec(command).stdout.on('data', (data) => {
-  stdoutCalls += 1;
-});
 
-exec('fhqwhgads').stderr.on('data', (data) => {
-  assert.strictEqual(typeof data, 'string');
-  stderrCalls += 1;
-});
+exec(command).stdout.on('data', common.mustCallAtLeast((data) => {
+}));
 
-process.on('exit', () => {
-  assert(stdoutCalls > 0);
-  assert(stderrCalls > 0);
-});
+exec('fhqwhgads').stderr.on('data', common.mustCall((data) => {
```

Comment:
I'd use `common.mustCallAtLeast()` here too, just to be safe because it's dealing with `data` events.

Reply:
@cjihrig Done!! Please have a look.